deep_quoridor/src/train_alphazero.py

```diff
@@ -50,19 +50,16 @@ def train_alphazero(
     for epoch in range(args.epochs):
         print(f"Starting epoch {epoch}")
 
-        # We use a different random seed each epoch to make sure we don't get correlated games, but
-        # we want the runs to be repeatable so we use a deterministic scheme to generate the seeds.
-        random_seed = args.seed + args.num_workers * epoch
-
         # When using per process evaluation, set the filename so that each process loads the most recent model.
         if args.per_process_evaluation:
-            self_play_params.model_filename = current_filename
+            self_play_params.model_filename = str(current_filename)
```

Comment:
This is just to keep the type-checker happy, since current_filename is a path but model_filename is a string.